app/helpers/podcast_planner_helper.rb

```diff
@@ -36,4 +40,32 @@ def time_options
 
     opts.map { |opt| [I18n.l(opt, format: :time_12_hour), opt] }
   end
+
+  def days_in_month(month)
+    month.map { |d| d.day }
+  end
+
+  def is_preselected_date?(date, dates)
+    if dates.present?
+      dates.include?(date)
+    else
+      false
+    end
+  end
+
+  def date_is_in_month?(date, month)
+    date.month == month
+  end
+
+  def calendar_day_tag(day:, month:, calendar:, &block)
```

Comment:
custom tag that modifies the `content_tag` from the gem generated partials. adds a `dateController` to dates within that month and avoids adding to "blank" spaces that are not within that month.